sci-libs/scalapack/scalapack-2.1.0.ebuild

```diff
@@ -3,11 +3,11 @@
 
 EAPI=7
 
-inherit cmake-utils fortran-2 toolchain-funcs
+inherit cmake fortran-2 toolchain-funcs
 
 DESCRIPTION="Subset of LAPACK routines redesigned for heterogenous (MPI) computing"
 HOMEPAGE="https://www.netlib.org/scalapack/"
-SRC_URI="https://www.netlib.org/scalapack/${P}.tgz"
+SRC_URI="https://chymera.eu/distfiles/${P}.tgz"
```

Comment:
Please host the patch in SRC_URI too (additional entry) rather than modified sources, it makes it easier to check etc.